src/pages/about.page.tsx

```diff
@@ -105,37 +105,34 @@ const AboutPage = () => {
               a customer.
             </p>
-            <p>
-              This includes:
-              <ul>
-                <li>public toilets</li>
-                <li>
-                  toilets in train stations, bus stations, service/petrol
-                  stations, tube stations, ferry terminals, airports and other
-                  transport networks
-                </li>
-                <li>shopping centre toilets</li>
-                <li>
-                  toilets in public buildings, such as town halls, libraries,
-                  hospitals, museums and leisure centres
-                </li>
-                <li>
-                  toilets in other businesses where the business agrees that the
-                  public can use their toilets without having to buy anything.
-                  These are often part of Community Toilet Schemes (run by
-                  councils) and can include shops, cafes, supermarkets,
-                  restaurants, hotels and pubs.
-                </li>
-                <li>
-                  We try not to show toilets where you need to ask permission to
-                  use it, such as those that are for customers-only. The
-                  exception is platform-side toilets at stations. These are only
-                  accessible to ticket-holders but we feel it is still useful
-                  information, particularly when many trains don’t have toilets
-                  onboard.
-                </li>
-              </ul>
-            </p>
+            <p>This includes: </p>
+            <ul>
+              <li>public toilets</li>
+              <li>
+                toilets in train stations, bus stations, service/petrol
+                stations, tube stations, ferry terminals, airports and other
+                transport networks
+              </li>
+              <li>shopping centre toilets</li>
+              <li>
+                toilets in public buildings, such as town halls, libraries,
+                hospitals, museums and leisure centres
+              </li>
+              <li>
+                toilets in other businesses where the business agrees that the
+                public can use their toilets without having to buy anything.
+                These are often part of Community Toilet Schemes (run by
+                councils) and can include shops, cafes, supermarkets,
+                restaurants, hotels and pubs.
+              </li>
+              <li>
+                We try not to show toilets where you need to ask permission to
+                use it, such as those that are for customers-only. The exception
+                is platform-side toilets at stations. These are only accessible
+                to ticket-holders but we feel it is still useful information,
+                particularly when many trains don’t have toilets onboard.
+              </li>
+            </ul>
```

Comment:
Is this diff intentional?

Reply:
Yes, that is a good question, I'd forgotten why I'd fixed that, but it was causing a react hydration error because of the illegal use of nesting a <ul> in a <p> tags. 😅 Turns out I don't know the HTML spec as well as I thought I did, thought I could sweep my error under the cover rather than admitting it in a commit 🙈

Reply:
Generally I'd suggest extracting these changes out to make it easier to review the PR, but not a blocker in this case.